examples/double-conversion/main.cpp

```diff
@@ -0,0 +1,6 @@
+#include <double-conversion.h>
```

Comment:
It should be `#include <double-conversion/double-conversion.h>`. If you will use `-I<root>/include/double-conversion` then you will pollute includes with internal headers. I.e. when user will do `#include <utils.h>` it will include `<root>/include/double-conversion/utils.h`.

Reply:
That makes sense. I was actually basing it on the generated Cmake config. But this is something I can change in a fork.

Reply:
> But this is something I can change in a fork.

I've created https://github.com/hunter-packages/double-conversion and added you to the team (check your email). You can push directly but it's better if you create pull request for review reasons. I've created branch `hunter-v3.0.0` from upstream tag `v3.0.0`.

Reply:
Thanks. I actually looked at the source and the headers themselves seem to include other local headers without using the directory path. For example, see https://github.com/google/double-conversion/blob/master/double-conversion/bignum.h#L31. I may have to fix these inclusions to fix the include directive.